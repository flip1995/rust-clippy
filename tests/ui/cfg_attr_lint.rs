#![warn(deprecated_cfg_attr)]

#![cfg_attr(feature="cargo-clippy", allow(absurd_extreme_comparisons))]

#[cfg_attr(feature="cargo-clippy", warn(absurd_extreme_comparisons, almost_swapped))]
fn main() {}
