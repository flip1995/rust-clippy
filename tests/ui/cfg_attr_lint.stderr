error: `cfg_attr` is deprecated for clippy and got replaced by tool_lints
 --> $DIR/cfg_attr_lint.rs:5:1
  |
5 | #[cfg_attr(feature="cargo-clippy", warn(absurd_extreme_comparisons, almost_swapped))]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: use: `#[warn(clippy::absurd_extreme_comparisons, clippy::almost_swapped)]`
  |
  = note: `-D deprecated-cfg-attr` implied by `-D warnings`

error: aborting due to previous error

